app/queries/servers/post.ts

```diff
@@ -101,11 +101,14 @@ export const queryPostsInThread = (database: Database, rootId: string, sorted =
     return database.get<PostsInThreadModel>(POSTS_IN_THREAD).query(...clauses);
 };
 
-export const queryPostReplies = (database: Database, rootId: string, excludeDeleted = true) => {
+export const queryPostReplies = (database: Database, rootId: string, excludeDeleted = true, excludeSystemMessages = false) => {
     const clauses: Q.Clause[] = [Q.where('root_id', rootId)];
     if (excludeDeleted) {
         clauses.push(Q.where('delete_at', Q.eq(0)));
     }
+    if (excludeSystemMessages) {
+        clauses.push(Q.where('type', ''));
```

Comment:
This should be not equal to system_message.

Also, I don't think `type` is indexed, so I want to know what is the performance impact of adding this to the query

Reply:
Something like `Q.where('type', Q.notLike('system_ephemeral%'))`?